NAS2D/Renderer/RendererOpenGL.cpp

```diff
@@ -445,15 +445,14 @@ void RendererOpenGL::drawBox(const Rectangle<float>& rect, Color color)
 		return;
 	}
 
-	glDisable(GL_TEXTURE_2D);
-
 	setColor(color);
+	glDisable(GL_TEXTURE_2D);
```

Comment:
Not sure why this is being moved.

Reply:
Normalize it with the drawBoxFilled function, purely a cosmetic change though I didn't mean to push it in this PR :D